fdbserver/workloads/IDDTxnProcessorApiCorrectness.actor.cpp

```diff
@@ -71,6 +71,7 @@ struct IDDTxnProcessorApiWorkload : TestWorkload {
 	std::string description() const override { return desc; }
 	Future<Void> setup(Database const& cx) override { return enabled ? _setup(cx, this) : Void(); }
 	Future<Void> start(Database const& cx) override { return enabled ? _start(cx, this) : Void(); }
+	void disableFailureInjectionWorkloads(std::set<std::string>& out) const override { out.insert("all"); }
```

Comment:
I understand why RandomMoveKeys is a problem, but this line disables all failure injections. Please either only add RandomMoveKeys or explain why this needs to disable everything.